rfc-154-integrating-an-ots-cms-with-govuk-for-custom-pages.md

```diff
@@ -0,0 +1,60 @@
+# Integrating an off-the-shelf CMS with GOV.UK infrastructure to power custom pages
+
+## Summary
+
+GOV.UK has built a number of bespoke content management systems (CMS) to enable the publishing of a variety of pre-defined document types. However, these tools are of limited assistance when a need emerges to create a one-off page on GOV.UK (hereafter referred to as custom page) that is not one of these pre-defined document types.
+
+Examples of these types of pages, and their content management solutions, are:
+
+- [GOV.UK Homepage](https://www.gov.uk/) and [GOV.UK Roadmap](https://www.gov.uk/roadmap), where Rails I18n has been used (and arguabably misused) as a basic [YAML CMS system](https://github.com/alphagov/frontend/blob/f2af829b14a9e6af21deb49c9bd0484e9ea49baa/config/locales/en.yml#L454-L576) which require developer support.
```

Comment:
I'd prefer that this part of the sentence be removed as it's kind of unhelpful, it violates the the [retrospective prime directive](https://www.funretrospectives.com/the-retrospective-prime-directive/), and we _know_ this anyway as otherwise this RFC wouldn't be being raised.

```suggestion
- [GOV.UK Homepage](https://www.gov.uk/) and [GOV.UK Roadmap](https://www.gov.uk/roadmap), where Rails I18n has been used as a basic [YAML CMS system](https://github.com/alphagov/frontend/blob/f2af829b14a9e6af21deb49c9bd0484e9ea49baa/config/locales/en.yml#L454-L576) which require developer support.
```

Reply:
Thanks for the feedback, I’ve edited the text. It wasn’t intended as a comment on individuals choices so it’s good to resolve that.